lib/feature_flagger/model.rb

```diff
@@ -12,10 +12,16 @@ def self.included(base)
       base.extend ClassMethods
     end
 
-    def rollout?(*feature_key)
+    def released?(*feature_key)
       self.class.released_id?(id, feature_key)
     end
 
+    # <b>DEPRECATED:</b> Please use <tt>release</tt> instead.
```

Comment:
This comment is wrong. Change `release` to `released?`, or just remove it all (Do we really need it?)

Reply:
Suggestion please!
![image](https://user-images.githubusercontent.com/817039/51555115-09b31080-1e5e-11e9-8e53-9f39b889b4ce.png)